dts/arm/st/g4/stm32g47X.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g4/stm32g4.dtsi>
+
+/ {
+	soc {
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020000 0x400>;
+			interrupts = <11 0 12 0 13 0 14 0 15 0 16 0 17 0 96 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
+			st,mem2mem;
+			dma-requests = <8>;
+			dma-offset = <0>;
+			status = "disabled";
+			label = "DMA_1";
+		};
+
+		dma2: dma@40020400 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020400 0x400>;
+			interrupts = <56 0 57 0 58 0 59 0 60 0 97 0 98 0 99 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x2>;
+			st,mem2mem;
+			dma-requests = <8>;
+			dma-offset = <8>;
+			status = "disabled";
+			label = "DMA_2";
+		};
+
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dma-cells = <4>;
+			reg = <0x40020800 0x400>;
+			interrupts = <94 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x4>;
+			dma-channels = <16>;
+			dma-generators = <4>;
+			dma-requests= <21>;
+			status = "disabled";
+			label = "DMAMUX_1";
+		};
```

Comment:
Why not adding these directly in stm32g4.dtsi ?

Reply:
I think I had a reason for that at the time, I will have to look into it again to see if it can be put in the general G4 file.

Reply:
@erwango they both have different values for the number of channels 12 vs 16.